test/images/agnhost/netexec/netexec.go

```diff
@@ -157,6 +161,18 @@ func (a *atomicBool) get() bool {
 
 func main(cmd *cobra.Command, args []string) {
 	exitCh := make(chan shutdownRequest)
+	stopCh := make(chan os.Signal, 1)
+	signal.Notify(stopCh, syscall.SIGTERM)
+
+	go func() {
+		<-stopCh
+		if terminationGracePeriodSeconds != 0 {
+			time.Sleep(time.Duration(terminationGracePeriodSeconds) * time.Second)
+		}
+
+		os.Exit(0)
```

Comment:
Contemplating this instead:
```go
exitCh <- shutdownRequest{
    code:    0,
    timeout: time.Second,
}
```

But it assumes that the exit handler is already running when SIGTERM is received, which we can't guarentee